include/render/gles2.h

```diff
@@ -60,31 +60,24 @@ struct wlr_gles2_renderer {
 	uint32_t viewport_width, viewport_height;
 };
 
-enum wlr_gles2_texture_type {
-	WLR_GLES2_TEXTURE_GLTEX,
-	WLR_GLES2_TEXTURE_WL_DRM_GL,
-	WLR_GLES2_TEXTURE_WL_DRM_EXT,
-	WLR_GLES2_TEXTURE_DMABUF,
-};
-
 struct wlr_gles2_texture {
 	struct wlr_texture wlr_texture;
-
 	struct wlr_egl *egl;
-	enum wlr_gles2_texture_type type;
-	int width, height;
-	bool has_alpha;
-	enum wl_shm_format wl_format; // used to interpret upload data
-	bool inverted_y;
 
-	// Not set if WLR_GLES2_TEXTURE_GLTEX
+	// Basically:
+	//   GL_TEXTURE_2D == mutable
+	//   GL_TEXTURE_EXTERNAL_OES == immutable
+	EGLint target;
```

Comment:
Nit: I guess this should be `GLint`

Reply:
You're right. I don't know why I used EGL there.
GLenum is an even more correct type.